Makefile

```diff
@@ -27,7 +27,7 @@ golint:
 	mkdir -p $(CURDIR)/.coverage
 
 test: .coverage
-	go test	-race -coverprofile=.coverage/coverage-unit.txt -covermode=atomic -v ./...
+	go test	-race -coverprofile=.coverage/coverage-unit.txt -covermode=atomic -run=^Test.*$ -v ./...
```

Comment:
Why do you need this change? With your change, you are skipping all of the following tests. Or am I overlooking something?
```
=== RUN   Test_parseAppFilter
=== RUN   Test_parseAppFilter/Correct_app_filter
=== RUN   Test_parseAppFilter/Correct_app_filter_with_deny
=== RUN   Test_parseAppFilter/Correct_app_filter_with_deny-all_policy
=== RUN   Test_parseAppFilter/Correct_app_filter_with_deny-all_policy_2
=== RUN   Test_parseAppFilter/Correct_app_filter_with_allow-all_policy
=== RUN   Test_parseAppFilter/Correct_app_filter_with_allow-all_policy_2
=== RUN   Test_parseAppFilter/incorrect_app_filter_bad_protocol
=== RUN   Test_parseAppFilter/incorrect_app_filter_bad_IP_format
=== RUN   Test_parseAppFilter/incorrect_app_filter_missing_precedence
=== RUN   Test_parseAppFilter/incorrect_app_filter_bad_precedence
--- PASS: Test_parseAppFilter (0.00s)
    --- PASS: Test_parseAppFilter/Correct_app_filter (0.00s)
    --- PASS: Test_parseAppFilter/Correct_app_filter_with_deny (0.00s)
    --- PASS: Test_parseAppFilter/Correct_app_filter_with_deny-all_policy (0.00s)
    --- PASS: Test_parseAppFilter/Correct_app_filter_with_deny-all_policy_2 (0.00s)
    --- PASS: Test_parseAppFilter/Correct_app_filter_with_allow-all_policy (0.00s)
    --- PASS: Test_parseAppFilter/Correct_app_filter_with_allow-all_policy_2 (0.00s)
    --- PASS: Test_parseAppFilter/incorrect_app_filter_bad_protocol (0.00s)
    --- PASS: Test_parseAppFilter/incorrect_app_filter_bad_IP_format (0.00s)
    --- PASS: Test_parseAppFilter/incorrect_app_filter_missing_precedence (0.00s)
    --- PASS: Test_parseAppFilter/incorrect_app_filter_bad_precedence (0.00s)
PASS
coverage: 5.3% of statements
ok      github.com/omec-project/pfcpsim/internal/pfcpsim        1.055s  coverage: 5.3% of statements
=== RUN   TestFARBuilderShouldPanic
=== RUN   TestFARBuilderShouldPanic/Invalid_FAR:_No_ID_provided
=== RUN   TestFARBuilderShouldPanic/Invalid_FAR:_Providing_TEID_without_DownlinkIP
=== RUN   TestFARBuilderShouldPanic/Invalid_FAR:_Providing_DownlinkIP_without_TEID
=== RUN   TestFARBuilderShouldPanic/Invalid_FAR:_Providing_both_forward_and_drop_actions
--- PASS: TestFARBuilderShouldPanic (0.00s)
    --- PASS: TestFARBuilderShouldPanic/Invalid_FAR:_No_ID_provided (0.00s)
    --- PASS: TestFARBuilderShouldPanic/Invalid_FAR:_Providing_TEID_without_DownlinkIP (0.00s)
    --- PASS: TestFARBuilderShouldPanic/Invalid_FAR:_Providing_DownlinkIP_without_TEID (0.00s)
    --- PASS: TestFARBuilderShouldPanic/Invalid_FAR:_Providing_both_forward_and_drop_actions (0.00s)
=== RUN   TestFARBuilder
=== RUN   TestFARBuilder/Valid_FAR
=== RUN   TestFARBuilder/Valid_FAR_action_with_2_flags
=== RUN   TestFARBuilder/Valid_FAR_actions_with_3_flags
--- PASS: TestFARBuilder (0.00s)
    --- PASS: TestFARBuilder/Valid_FAR (0.00s)
    --- PASS: TestFARBuilder/Valid_FAR_action_with_2_flags (0.00s)
    --- PASS: TestFARBuilder/Valid_FAR_actions_with_3_flags (0.00s)
=== RUN   TestPDRBuilderShouldPanic
=== RUN   TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_No_ID_provided
=== RUN   TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_Partial_parameters_provided
=== RUN   TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_Partial_parameters_provided#01
=== RUN   TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_build_without_MarkAsDownlink
=== RUN   TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_marked_as_uplink_passing_downlink_parameters
--- PASS: TestPDRBuilderShouldPanic (0.00s)
    --- PASS: TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_No_ID_provided (0.00s)
    --- PASS: TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_Partial_parameters_provided (0.00s)
    --- PASS: TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_Partial_parameters_provided#01 (0.00s)
    --- PASS: TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_build_without_MarkAsDownlink (0.00s)
    --- PASS: TestPDRBuilderShouldPanic/Invalid_Downlink_PDR:_marked_as_uplink_passing_downlink_parameters (0.00s)
=== RUN   TestPDRBuilder
=== RUN   TestPDRBuilder/Valid_Create_Uplink_PDR
=== RUN   TestPDRBuilder/Valid_Create_Downlink_PDR
=== RUN   TestPDRBuilder/Valid_Update_Downlink_PDR_no_SDF
=== RUN   TestPDRBuilder/Valid_Delete_Downlink_PDR
--- PASS: TestPDRBuilder (0.00s)
    --- PASS: TestPDRBuilder/Valid_Create_Uplink_PDR (0.00s)
    --- PASS: TestPDRBuilder/Valid_Create_Downlink_PDR (0.00s)
    --- PASS: TestPDRBuilder/Valid_Update_Downlink_PDR_no_SDF (0.00s)
    --- PASS: TestPDRBuilder/Valid_Delete_Downlink_PDR (0.00s)
=== RUN   TestQERBuilderShouldPanic
=== RUN   TestQERBuilderShouldPanic/Invalid_QER:_No_ID_provided
--- PASS: TestQERBuilderShouldPanic (0.00s)
    --- PASS: TestQERBuilderShouldPanic/Invalid_QER:_No_ID_provided (0.00s)
=== RUN   TestQERBuilder
=== RUN   TestQERBuilder/Valid_Create_QER_with_gate_open
=== RUN   TestQERBuilder/Valid_Create_QER_with_Gate_closed
=== RUN   TestQERBuilder/Valid_Update_QER
=== RUN   TestQERBuilder/Valid_Delete_QER
--- PASS: TestQERBuilder (0.00s)
    --- PASS: TestQERBuilder/Valid_Create_QER_with_gate_open (0.00s)
    --- PASS: TestQERBuilder/Valid_Create_QER_with_Gate_closed (0.00s)
    --- PASS: TestQERBuilder/Valid_Update_QER (0.00s)
    --- PASS: TestQERBuilder/Valid_Delete_QER (0.00s)
PASS
coverage: 90.6% of statements
```

Reply:
Moreover, with your change above, you are skipping all of the FUZZ tests and, as you can see below, all of them fail:

```
...
...
--- FAIL: Fuzz (500.98s)
    --- FAIL: Fuzz/seed#0 (5.01s)
    --- FAIL: Fuzz/seed#1 (5.01s)
    --- FAIL: Fuzz/seed#2 (5.01s)
    --- FAIL: Fuzz/seed#3 (5.01s)
    --- FAIL: Fuzz/seed#4 (5.02s)
    --- FAIL: Fuzz/seed#5 (5.01s)
    --- FAIL: Fuzz/seed#6 (5.01s)
    --- FAIL: Fuzz/seed#7 (5.01s)
    --- FAIL: Fuzz/seed#8 (5.01s)
    --- FAIL: Fuzz/seed#9 (5.01s)
    --- FAIL: Fuzz/seed#10 (5.01s)
    --- FAIL: Fuzz/seed#11 (5.01s)
    --- FAIL: Fuzz/seed#12 (5.01s)
    --- FAIL: Fuzz/seed#13 (5.01s)
    --- FAIL: Fuzz/seed#14 (5.01s)
    --- FAIL: Fuzz/seed#15 (5.01s)
    --- FAIL: Fuzz/seed#16 (5.02s)
    --- FAIL: Fuzz/seed#17 (5.01s)
    --- FAIL: Fuzz/seed#18 (5.01s)
    --- FAIL: Fuzz/seed#19 (5.01s)
    --- FAIL: Fuzz/seed#20 (5.01s)
    --- FAIL: Fuzz/seed#21 (5.01s)
    --- FAIL: Fuzz/seed#22 (5.01s)
    --- FAIL: Fuzz/seed#23 (5.01s)
    --- FAIL: Fuzz/seed#24 (5.01s)
    --- FAIL: Fuzz/seed#25 (5.01s)
    --- FAIL: Fuzz/seed#26 (5.01s)
    --- FAIL: Fuzz/seed#27 (5.01s)
    --- FAIL: Fuzz/seed#28 (5.01s)
    --- FAIL: Fuzz/seed#29 (5.01s)
    --- FAIL: Fuzz/seed#30 (5.01s)
    --- FAIL: Fuzz/seed#31 (5.01s)
    --- FAIL: Fuzz/seed#32 (5.01s)
    --- FAIL: Fuzz/seed#33 (5.00s)
    --- FAIL: Fuzz/seed#34 (5.01s)
    --- FAIL: Fuzz/seed#35 (5.01s)
    --- FAIL: Fuzz/seed#36 (5.01s)
    --- FAIL: Fuzz/seed#37 (5.01s)
    --- FAIL: Fuzz/seed#38 (5.00s)
    --- FAIL: Fuzz/seed#39 (5.00s)
    --- FAIL: Fuzz/seed#40 (5.01s)
    --- FAIL: Fuzz/seed#41 (5.00s)
    --- FAIL: Fuzz/seed#42 (5.01s)
    --- FAIL: Fuzz/seed#43 (5.01s)
    --- FAIL: Fuzz/seed#44 (5.01s)
    --- FAIL: Fuzz/seed#45 (5.01s)
    --- FAIL: Fuzz/seed#46 (5.01s)
    --- FAIL: Fuzz/seed#47 (5.01s)
    --- FAIL: Fuzz/seed#48 (5.01s)
    --- FAIL: Fuzz/seed#49 (5.01s)
    --- FAIL: Fuzz/seed#50 (5.01s)
    --- FAIL: Fuzz/seed#51 (5.01s)
    --- FAIL: Fuzz/seed#52 (5.01s)
    --- FAIL: Fuzz/seed#53 (5.01s)
    --- FAIL: Fuzz/seed#54 (5.01s)
    --- FAIL: Fuzz/seed#55 (5.01s)
    --- FAIL: Fuzz/seed#56 (5.01s)
    --- FAIL: Fuzz/seed#57 (5.01s)
    --- FAIL: Fuzz/seed#58 (5.01s)
    --- FAIL: Fuzz/seed#59 (5.01s)
    --- FAIL: Fuzz/seed#60 (5.01s)
    --- FAIL: Fuzz/seed#61 (5.01s)
    --- FAIL: Fuzz/seed#62 (5.01s)
    --- FAIL: Fuzz/seed#63 (5.01s)
    --- FAIL: Fuzz/seed#64 (5.01s)
    --- FAIL: Fuzz/seed#65 (5.01s)
    --- FAIL: Fuzz/seed#66 (5.01s)
    --- FAIL: Fuzz/seed#67 (5.01s)
    --- FAIL: Fuzz/seed#68 (5.01s)
    --- FAIL: Fuzz/seed#69 (5.01s)
    --- FAIL: Fuzz/seed#70 (5.01s)
    --- FAIL: Fuzz/seed#71 (5.01s)
    --- FAIL: Fuzz/seed#72 (5.01s)
    --- FAIL: Fuzz/seed#73 (5.01s)
    --- FAIL: Fuzz/seed#74 (5.01s)
    --- FAIL: Fuzz/seed#75 (5.01s)
    --- FAIL: Fuzz/seed#76 (5.01s)
    --- FAIL: Fuzz/seed#77 (5.01s)
    --- FAIL: Fuzz/seed#78 (5.01s)
    --- FAIL: Fuzz/seed#79 (5.01s)
    --- FAIL: Fuzz/seed#80 (5.01s)
    --- FAIL: Fuzz/seed#81 (5.01s)
    --- FAIL: Fuzz/seed#82 (5.01s)
    --- FAIL: Fuzz/seed#83 (5.01s)
    --- FAIL: Fuzz/seed#84 (5.01s)
    --- FAIL: Fuzz/seed#85 (5.01s)
    --- FAIL: Fuzz/seed#86 (5.01s)
    --- FAIL: Fuzz/seed#87 (5.01s)
    --- FAIL: Fuzz/seed#88 (5.01s)
    --- FAIL: Fuzz/seed#89 (5.01s)
    --- FAIL: Fuzz/seed#90 (5.01s)
    --- FAIL: Fuzz/seed#91 (5.01s)
    --- FAIL: Fuzz/seed#92 (5.01s)
    --- FAIL: Fuzz/seed#93 (5.01s)
    --- FAIL: Fuzz/seed#94 (5.01s)
    --- FAIL: Fuzz/seed#95 (5.01s)
    --- FAIL: Fuzz/seed#96 (5.01s)
    --- FAIL: Fuzz/seed#97 (5.01s)
    --- FAIL: Fuzz/seed#98 (5.01s)
    --- FAIL: Fuzz/seed#99 (5.01s)
FAIL
coverage: 2.5% of statements
FAIL    github.com/omec-project/pfcpsim/fuzz    501.051s
...
...
```

Reply:
Hi @gab-arrobo 
Thanks for your comment.
The fuzz test here is an integration test, it needs work with the running UPF.
So I want to ignore the fuzz test in the CI/CD pipeline.

Reply:
Can you use a `mock` UPF for this test?

Reply:
Hi @gab-arrobo 

The purpose of fuzz here is to make the tested UPF crash (If it can not handle the malformed PFCP message).
So I think we don't need to mock the UPF instance.
Tks!